dashboard/app/models/script.rb

```diff
@@ -744,6 +744,17 @@ def summarize
     summary
   end
 
+  # Similar to summarize, but returns an even more narrow set of fields, in particular
+  # bypasses stage summaries
+  def summarize_short
```

Comment:
I know this gets hit in tests, but it'd be nice to have a new unit test pinning just the behavior of `summarize_short`, since it's new public API for this model.